package.json

```diff
@@ -60,6 +60,7 @@
                         "request": "launch",
                         "platform": "android",
                         "internalDebuggerPort": 9090,
+                        "extensionServerPort": 8099,
```

Comment:
Note that the "default" specified here is only for documentation. We need to ensure that our code copes with missing values and explicitly set a default.